### DISCUSSION TITLE
IP leaks

### DISCUSSION THREAD

**Discussion by skkim99** [geckosio/geckos.io#287] [Help / Questions]
Can your public IP get leaked to others when webrtc is used in a client-server model?

**Comment by reececomo** [comment]
#### For server IP address
Yes, the server's public IP address will be exposed (and this is desirable).

#### For client IP addresses

Yes and no.

- No, its not exposed to other clients by Geckos.io.
- But also yes, its _technically_ possible for public IP to be leaked, by you (e.g. logging, accidentally broadcasting them to other clients, etc.).
